src/components/RecommendationCount/RecommendationCount.tsx

```diff
@@ -2,14 +2,17 @@ import React, { useState } from "react";
 import { css } from "emotion";
 
 import { textSans } from "@guardian/src-foundations/typography";
+import { neutral } from "@guardian/src-foundations/palette";
 import { palette } from "@guardian/src-foundations";
 
 import { recommend } from "../../lib/api";
+import { Pillar } from "../../types";
 
 type Props = {
   commentId: number;
   initialCount: number;
   alreadyRecommended: boolean;
+  pillar: Pillar;
```

Comment:
Recommendation doesn't use pillar for background colour, it's always this blue. I think you can grab it from the `brand` palette

![Screenshot 2020-04-02 at 18 25 22](https://user-images.githubusercontent.com/1336821/78279417-63f23b80-750f-11ea-983a-fca4793e96b2.jpg)